### PR TITLE
Tests for wide combiner with spilling

### PR DESCRIPTION
**Pull request by lll-phill-lll**
### Changelog entry 

Added tests for wide combiner with spilling
...

### Changelog category 

* Not for changelog (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp [hunk 1]
```diff
@@ -722,9 +733,7 @@ class TSpillingSupportState : public TComputationValue<TSpillingSupportState> {
     }
 
     bool IsSwitchToSpillingModeCondition() const {
-        return false;
-        // TODO: YQL-18033
-        // return !HasMemoryForProcessing();
+        return !HasMemoryForProcessing();
```
Comment (lll-phill-lll) [inline ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp:736]: Please note that the hardcoded value is removed here

File: ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp [hunk 2]
```diff
@@ -564,6 +559,19 @@ class TSpillingSupportState : public TComputationValue<TSpillingSupportState> {
...
+        const auto used = TlsAllocState->GetUsed();
+        const auto limit = TlsAllocState->GetLimit();
+        TStringBuilder logmsg;
+        logmsg << "Memory usage: ";
+        if (limit) {
+            logmsg << (used*100/limit) << "%=";
+        }
+        logmsg << (used/1_MB) << "MB/" << (limit/1_MB) << "MB";
+
+        YQL_LOG(INFO) << logmsg;
```
Comment (Darych) [inline ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp:572]: Maybe we want to print it on Debug level?

File: ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp [hunk 3]
```diff
@@ -688,10 +696,12 @@ class TSpillingSupportState : public TComputationValue<TSpillingSupportState> {
     void SwitchMode(EOperatingMode mode) {
         switch(mode) {
             case EOperatingMode::InMemory: {
+                YQL_LOG(INFO) << "switching Memory mode to InMemory";
```
Comment (Darych) [inline ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp:699]: And here also in Debug?

**Review by Darych — APPROVED** [review]

File: ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp (hunk 3 above)
Comment (lll-phill-lll) [reply ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp:699]: Discussed that this log would be written once per task, so we decided to leave the INFO level.

File: ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp (hunk 2 above)
Comment (lll-phill-lll) [reply ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp:572]: Discussed that this log would be written once per task, so we decided to leave the INFO level.

**Pull request merged**